packages/core/src/State.ts

```diff
@@ -302,6 +302,7 @@ export function getPersistedState<TState extends AnyState>(
 
   return {
     ...jsonValues,
-    children: childrenJson
+    children: childrenJson,
+    actions: []
```

Comment:
shouldn't we also discard those in `StateMachine['restoreState']`? This would require us to change those two tests:
- [`interpreter.test.ts`: executes actions from a restored state](https://github.com/statelyai/xstate/blob/297cd61cef31d19dda3e58e2c76ba7250461d5b0/packages/core/test/interpreter.test.ts#L110-L158)
- [`after.test.ts`: should execute an after transition after starting from a state resolved using `machine.getInitialState`](https://github.com/statelyai/xstate/blob/297cd61cef31d19dda3e58e2c76ba7250461d5b0/packages/core/test/after.test.ts#L140-L159)

Reply:
Yeah, we can do that

Reply:
👋

Reply:
I did that